logger/logger.go

```diff
@@ -80,7 +81,10 @@ func useTestLogFile(args []string) bool {
 // GetLogFileLocation will return the location on disk of the log file
 func GetLogFileLocation() (result string) {
 	result, _ = os.UserHomeDir()
-	os.MkdirAll(path.Join(result, types.OssIndexDirName), os.ModePerm)
+	err := os.MkdirAll(path.Join(result, types.OssIndexDirName), os.ModePerm)
+	if err != nil {
+		fmt.Print(err)
```

Comment:
is that error going to make sense in isolation?

Reply:
Yeah it wasn't great...followed the other error print in here. So I now fixed them both in 124d008